### DISCUSSION TITLE
Is passphrase rotation supported for encrypted volume ?

### DISCUSSION THREAD

**Discussion by sushiMix** [longhorn/longhorn#9342] [Q&A] [answered]
Hello, I see in LUKS documentation that we are able to rotate passphrase for encrypted volume, but i can't see any clue about it in documentation.
Is it available in longhorn ?

**Comment by derekbit** [comment, marked as answer]
Longhorn doesn't support it for now, so this is a feature request.

cc @ChanYiLin @mantissahz @innobead

**Comment by kesavkolla** [comment]
Any movemove on this? We do encrypt the entire database volume on longhorn encrypted volume. We are currently unable to implement keyrotation policy. This is importatnt feature to implement from security standpoint.

**Reply by derekbit** [reply]
@kesavkolla 
Not in Longhorn's roadmap for now. Please create a feature request in https://github.com/longhorn/longhorn/issues. Thanks.
